ncm-dirperm/src/main/pan/components/dirperm/schema.pan

```diff
@@ -30,11 +30,13 @@ function dirperm_permissions_valid = {
 };
 
 type structure_dirperm_entry = {
-    'path'    : string with match(SELF, '^/')
-    'perm'    : string with match(SELF,'[0-7]{3,4}')
-    'owner'   : string with match(SELF, '\w+(:\w+)?')
-    'type'    : string with match(SELF, 'd|f')
-    'initdir' ? string[]
+    'path'       : string with match(SELF, '^/')
+    'perm'       : string with match(SELF,'[0-7]{3,4}')
+    'owner'      : string with match(SELF, '\w+(:\w+)?')
+    'type'       : string with match(SELF, 'd|f')
+    'initdir'    ? string[]
+    'checkmount' : boolean = false
+    'notroot'    : boolean = false
```

Comment:
i don't understand. this is a new attribute, so it won't affect anyone how it is named or defined. `notroot=false` and `root=true` are the same thing. (and i meant to change it here and invert the logic in the code as well ofcourse).